cmd/extend.go

```diff
@@ -0,0 +1,19 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+var extendCmd = &cobra.Command{
+	Use:   "extend",
+	Short: "Extends a previously generated schedule",
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Println("Extending schedule")
```

Comment:
Guessing we would fill this out with something like the `RunE:  execute` of `generate.go`?

Reply:
Yep, that's the intent - `Run` and `RunE` are the same, with `RunE` simply returning an `error`